app/Http/Controllers/Admin/UserController.php

```diff
@@ -45,7 +47,17 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
+        $data = $request->all();
+        if ($request->hasFile('avatar')) {
+            $request->avatar->store(config('site.user.image'));
+
+            $data['avatar'] = $request->avatar->hashName();
+        }
+       $data['password'] = bcrypt($request->password);
```

Comment:
sao ko dùng mutator nhỉ https://laravel.com/docs/5.7/eloquent-mutators#accessors-and-mutators

Reply:
vâng, để em fix ạ